wsd/ClientSession.cpp

```diff
@@ -2189,14 +2189,18 @@ bool ClientSession::handleKitToClientMessage(const std::shared_ptr<Message>& pay
         else if (tokens.equals(0, "extractedlinktargets:"))
         {
             LOG_TRC("Sending extracted link targets response.");
+            if (!_saveAsSocket)
```

Comment:
One should read these with 'git show -w' and then these are straightforward modifications.